loopd/swapclient_server.go

```diff
@@ -101,6 +101,13 @@ func (s *swapClientServer) LoopOut(ctx context.Context,
 
 	log.Infof("Loop out request received")
 
+	paymentTimeout := time.Duration(in.PaymentTimeout) * time.Second
+	// Make sure we don't exceed the total allowed payment timeout.
+	if paymentTimeout > s.config.TotalPaymentTimeout {
```

Comment:
I propose to add a comment that `in.PaymentTimeout` is uint32, so it is >= 0 (no need to check that separately).